### PR TITLE
Update use-upgrade-readiness-to-manage-windows-upgrades.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/3345
Added important note.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/upgrade/use-upgrade-readiness-to-manage-windows-upgrades.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ ms.topic: article
 
 # Use Upgrade Readiness to manage Windows upgrades
 
+>[!IMPORTANT]
+>**The OMS portal has been deprecated; you should start using the [Azure portal](https://portal.azure.com) instead as soon as possible.** Many experiences are the same in the two portals, but there are some key differences. See [Windows Analytics in the Azure Portal](../update/windows-analytics-azure-portal.md) for steps to use Windows Analytics in the Azure portal. For much more information about the transition from OMS to Azure, see [OMS portal moving to Azure](https://docs.microsoft.com/azure/log-analytics/log-analytics-oms-portal-transition). [Get started with Upgrade Readiness](https://docs.microsoft.com/windows/deployment/upgrade/upgrade-readiness-get-started)
```
Comment (JohanFreelancer9) [inline windows/deployment/upgrade/use-upgrade-readiness-to-manage-windows-upgrades.md]:
```suggestion
>**The OMS portal has been deprecated; you should start using the [Azure portal](https://portal.azure.com) instead as soon as possible.** Many experiences are the same in the two portals, but there are some key differences. See [Windows Analytics in the Azure Portal](../update/windows-analytics-azure-portal.md) for steps to use Windows Analytics in the Azure portal. For much more information about the transition from OMS to Azure, see [OMS portal moving to Azure](https://docs.microsoft.com/azure/log-analytics/log-analytics-oms-portal-transition) and [Get started with Upgrade Readiness](https://docs.microsoft.com/windows/deployment/upgrade/upgrade-readiness-get-started).
```

Comment (nenonix) [inline windows/deployment/upgrade/use-upgrade-readiness-to-manage-windows-upgrades.md]:
```suggestion
>**The OMS portal has been deprecated. Switch to the [Azure portal](https://portal.azure.com) instead.** The two portals offer the same experience, with some key differences. Learn how to use [Windows Analytics in the Azure Portal]((../update/windows-analytics-azure-portal.md) Find out more about the [OMS portal moving to Azure](https://docs.microsoft.com/azure/log-analytics/log-analytics-oms-portal-transition) or jump right in and [Get started with Upgrade Readiness](https://docs.microsoft.com/windows/deployment/upgrade/upgrade-readiness-get-started).
```

**Comment by mypil** [comment]
@Reeced40 - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

File: windows/deployment/upgrade/use-upgrade-readiness-to-manage-windows-upgrades.md [hunk 2]
```diff
@@ -12,6 +12,9 @@ ms.topic: article
 
 # Use Upgrade Readiness to manage Windows upgrades
 
+>[!IMPORTANT]
+>**The OMS portal has been deprecated; you should start using the [Azure portal](https://portal.azure.com) instead as soon as possible.** Many experiences are the same in the two portals, but there are some key differences. See [Windows Analytics in the Azure Portal](../update/windows-analytics-azure-portal.md) for steps to use Windows Analytics in the Azure portal. For much more information about the transition from OMS to Azure, see [OMS portal moving to Azure](https://docs.microsoft.com/azure/log-analytics/log-analytics-oms-portal-transition) and [Get started with Upgrade Readiness](https://docs.microsoft.com/windows/deployment/upgrade/upgrade-readiness-get-started).
```
Comment (nenonix) [inline windows/deployment/upgrade/use-upgrade-readiness-to-manage-windows-upgrades.md]:
```suggestion
>>**The OMS portal has been deprecated, so you need to switch to the [Azure portal](https://portal.azure.com) now.** The two portals offer the same experience, with some key differences. Learn how to use [Windows Analytics in the Azure Portal](../update/windows-analytics-azure-portal.md). Find out more about the [OMS portal moving to Azure](https://docs.microsoft.com/azure/log-analytics/log-analytics-oms-portal-transition), or jump right in and [Get started with Upgrade Readiness](https://docs.microsoft.com/windows/deployment/upgrade/upgrade-readiness-get-started).
```

**Comment by mypil** [comment]
@jaimeo - Can you please assist to check, approve and merge the content for this PR based on issue #3345?

Thank you.

**Pull request merged**